src/base/sign-in-off/_base.scss

```diff
@@ -0,0 +1,8 @@
+#wb-so {
+	text-align: $wb-so-text-align;
+
+	.row {
+		background: $wb-so-background-color;
+		padding: 1em 0 0;
```

Comment:
Maybe there'd be value in removing ``padding: 1em 0 0;``? The 1em of top padding produces a pretty large space above the buttons. That space doesn't currently exist.